code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -1,5 +1,4 @@
 /mob/living/carbon/human/proc/do_quick_equip()
-	SIGNAL_HANDLER_DOES_SLEEP
```

Comment:
Do not remove signal handlers

Reply:
but you said that handler. was "illegal" sooner

Reply:
I said adding new sleeping ones is not allowed 
do not remove signal handlers